src/simple-datetime.ts

```diff
@@ -1,5 +1,6 @@
 import * as vscode from 'vscode';
 import * as datefns from 'date-fns';
+import * as Utils from './utils';
```

Comment:
project convention for `import` statements seems to be to use lowercase, is there a typescript convention for when to use `Utils` vs `utils`?

Reply:
Both subtree-functions.ts and timestamp-functions.ts have Utils instead of utils, that's why I kept the uppercase.

Reply:
👍